lib/katello/permissions/sync_plan_permissions.rb

```diff
@@ -4,7 +4,8 @@
   permission :view_sync_plans,
              {
                'katello/sync_plans' => [:all, :index, :auto_complete_search],
-               'katello/api/v2/sync_plans' => [:index, :show, :add_products, :remove_products, :available_products]
+               'katello/api/v2/sync_plans' => [:index, :show, :add_products, :remove_products],
```

Comment:
Since we didn't get rid of the route as of this change, do not still want it protected by a permission?